gensim/corpora/__init__.py

```diff
@@ -1,17 +1,17 @@
 """
 This package contains implementations of various streaming corpus I/O format.
 """
-
+# flake8: noqa
```

Comment:
This seems to combine the worst of both worlds. Still extra comments, AND no style checking.

I think I prefer even the `noqa:F401` version.